lib/dependencies.js

```diff
@@ -46,7 +46,10 @@ let versionMap = {
   "minimatch": "^3.0.2",
   "through2": "^2.0.1",
   "tslint": "^3.11.0",
-  "typings": "^1.3.0",
+  "@types/protractor": "^1.5.0",
+  "@types/jasmine": "^2.2.0",
+  "@types/selenium-webdriver": "^2.44.0",
```

Comment:
Protractor 4 comes with its own typings and they contain jasmine and selenium. No need to even add these three nowadays. The rest is okay.